mitk-plugin/Modules/NvidiaAIAAModule/src/NvidiaDextrSegTool3D.cpp

```diff
@@ -590,43 +603,42 @@ void NvidiaDextrSegTool3D::ItkImageProcessDextr3D(itk::Image<TPixel, VImageDimen
 
     // Do Sampling
     nvidia::aiaa::ImageInfo imageInfo;
-    nvidia::aiaa::Pixel::Type pixelType = nvidia::aiaa::getPixelType(demangle(typeid(TPixel).name()));
-    MITK_INFO("nvidia") << "++++ typeid(TPixel): " << typeid(TPixel).name() << "; demangle(typeid(TPixel)): " << demangle(typeid(TPixel).name())
-        << "; PixelType: " << nvidia::aiaa::getPixelTypeStr(pixelType);
-    MITK_INFO("nvidia") << "++++ VImageDimension: " << VImageDimension;
-
     LATENCY_START_API_CALL()
-    nvidia::aiaa::PointSet pointSetROI = imagePreProcess(pointSet, itkImage, tmpSampleFileName, imageInfo, model.padding, model.roi);
-    //void* inputImage = itkImage;
-    //client.sampling(model, pointSet, inputImage, pixelType, VImageDimension, tmpSampleFileName, imageInfo);
 
+    nvidia::aiaa::PointSet pointSetROI = imagePreProcess(pointSet, itkImage, tmpSampleFileName, imageInfo, model.padding, model.roi);
```

Comment:
should this be "nvidia::aiaa::AiaaUtils::imagePreProcess" follows line 626?
or why not?

Reply:
Small issue here.. we can't use nvidia::aiaa::AiaaUtils::imagePreProcess
For that MITK has to save the image locally and then call.. which takes more time today...

so to make it faster, we have copied the functionality inside MITK plugin.. so that it can operate directly on MitkImage * and save only cropped image (as part of pre-process)